frontend/dispatcher.lua

```diff
@@ -312,8 +312,10 @@ function Dispatcher:getNameFromItem(item, location, settings)
     if location[settings] ~= nil and location[settings][item] ~= nil then
         amount = location[settings][item]
     end
-    if amount == nil or amount == 0 then
-        amount = "X"
+    if amount == nil
+        or (amount == 0 and settingsList[item].category == "incrementalnumber")
+    then
+        amount = "#"
```

Comment:
> # as a placeholder for a number may be less common outside of the US, but it's fine by me ;).

Just leave it to the translators. ;-)
```suggestion
        amount = C_("Number placeholder", "#")
```

Reply:
Huh, that's cool :).